### PR TITLE
Fix creating token with null bytes on name/symbol

### PR DESCRIPTION
**Pull request by Uxio0**
- Fix creating token with null bytes on name/symbol
- Set version 4.2.6

### CODE REVIEW

**Review by jpalvarezl — APPROVED** [review]
Looks good 🙂

File: safe_transaction_service/tokens/tests/test_models.py [hunk 1]
```diff
@@ -157,3 +157,22 @@ def test_create_from_blockchain_with_byte_data(self, get_info: MagicMock):
         token = Token.objects.get(address="0xBB9bc244D798123fDe783fCc1C72d3Bb8C189413")
         self.assertEqual(token.name, "�������������������������������\x01")
         self.assertEqual(token.symbol, "�������������������������������\x01")
+
+    @mock.patch.object(
+        Erc20Manager,
+        "get_info",
+        autospec=True,
+        return_value=Erc20Info(
+            name="Build\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00",
```
Comment (rmeissner) [inline safe_transaction_service/tokens/tests/test_models.py:166]: if this should be handled like a c-type memory for strings, then 0-bytes terminate the string. Therefore I would not replace them but look for the first 0-byte and cut of everything afterwards

Comment (Uxio0) [reply safe_transaction_service/tokens/tests/test_models.py:166]: Yeah, but are we sure `0-bytes` are not allowed anywhere else?

**Comment by Uxio0** [comment]
@rmeissner PR was updated with more cases and always ignoring and logging an error inserting a token so we don't 500 again

**Comment by Uxio0** [comment]
I will merge it as I'm seeing 500 in production and this should fix them

**Pull request merged**